src/iio_wrapper.cpp

```diff
@@ -100,6 +100,48 @@ IIOWrapper::IIOWrapper()
         m_local_context = iio_create_local_context();
         if(!m_local_context)
             throw std::runtime_error("Exception: local context is null");
+
+        int count = iio_context_get_devices_count(m_local_context);
+        if(count>0)
+        {
+            struct iio_device *dev = iio_context_get_device(m_local_context, 0);
+            const char *name = iio_device_get_name(dev);
+            RCLCPP_INFO(rclcpp::get_logger("rclcpp_device_name"), "device name: %s", name);
+
+            IIOWrapper::s_device_name = std::string(name);
+
+            std::list<std::string> adis16505_x;
```

Comment:
Small suggestion: initialization might be easier with:
std::list<std::string> adis16505_x {"adis16505", "adis16505-1", ...}